test/ctia/test_helpers/aggregate.clj

```diff
@@ -219,10 +223,15 @@
   (let [enumerable-schema (schema-enumerable-fields new-schema enumerable-fields)
         base-doc (dissoc entity-minimal :id)]
     (doall
-     (repeatedly n (fn [] (merge base-doc
-                                 (g/generate enumerable-schema
-                                             {s/Str string-generator})
-                                 (generate-date-fields date-fields)))))))
+     (repeatedly n (fn [] (deep-merge-with
+                          (fn [a b]
+                            (if (seq? a)
```

Comment:
Not necessarily wrong (hard for me to tell without diving into the code), but `seq?` sometimes has an intent that would be more explicit with some other pred (`seq`, `coll?`, `list?`, `sequential?`)